README.md

```diff
@@ -2,24 +2,220 @@
+
+The ASC Co-Chairs shall check the status of discussion on the PR on a weekly basis. Once discussion appears to have consolidated (either measured by participation in the straw poll, rate of comments, or a combination of both), the Co-Chairs will schedule the proposal for a reading at the next quarterly meeting of the ASC. The proposal shall be included in the announced agenda for the meeting along with a link to the PR and the results of the straw poll.
+
+At least one proposal author must attend the quarterly meeting to present the PR and participate in discussion regarding its approval. A "reading" of the proposal shall be conducted at the meeting with the author(s) reviewing any questions/concerns raised and how they were addressed. During the presentation, someone designated by the Co-Chairs will take notes on the PR (and/or Issue) on behalf of the presenter. At the end of the agenda item, those members wishing to adjust their straw votes will be given an opportunity to do so. Members who did not attend the meeting will also have a chance to adjust their votes based on the published minutes of the meeting.
+
+The final straw vote results shall be tallied no later than two weeks following the quarterly meeting where the proposal was presented. At that time, the Co-Chairs shall set a GitHub Label corresponding to the result - either indicating "Accepted" or "Pushed Back" based on the final results of the "**Straw Poll**". Accepted proposals can be committed by the Release Manager in accordance with their schedule. Proposals that are pushed back for further work can be brought forward again at a later date, or can be withdrawn by the author(s) by simply closing the PR.
+
+
+#### Elevation to Stable status
+Provisional elements of the Standard that have been included in at least two minor releases become eligible for elevation to "**Stable**" status. The elevation process begins with a PR requesting consideration of the change, identifying the specific elements (APIs and attributes) to be included, and a justification for Stable status. The PR is opened in a [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) state until ready for submission for review. Justification should be commensurate with the magnitude of the request - e.g., elevating an API to "Stable" likely requires more justification than a key-value attribute. While there is no clear formula for justifying elevation, typical considerations might include the breadth of adoption (by implementers, system management vendors, and/or programming libraries) and stability of the elements themselves (i.e., time since last modification).
```

Comment:
I do not think so.  At least, if we follow semantic versioning, a major version change is only required when a breaking API change is made.  If it is required, then it may be years between when an interface goes from "Provisional" to "Stable".  That seems unnecessarily long to me

I do think that it requires a minor version change.

Reply:
Though each elevation can be reviewed independently with respect to a major version bump, it feels like going from Provisional to Stable changes the strength of the API/key-value which breaks backward compatibility.
So I thought there might be a general rule about that.